internal/util/k8sutil/crd.go

```diff
@@ -62,13 +63,18 @@ func GetCRDManifestPaths(crdsDir string) (crdPaths []string, err error) {
 			if err != nil {
 				return fmt.Errorf("error reading manifest %s: %v", path, err)
 			}
+			// Skip files in crdsDir that aren't k8s manifests since we do not know
+			// what other files are in crdsDir.
 			typeMeta, err := GetTypeMetaFromBytes(b)
 			if err != nil {
-				return fmt.Errorf("error getting kind from manifest %s: %v", path, err)
+				log.Debugf("Skipping non-manifest file %s: %v", path, err)
```

Comment:
Are you adding these debug statements for testing or with the intent of permanence?

Reply:
These ones are intentional. These will only show up when `generate csv` is run with `--verbose` mode. And in that case it can be useful to see what files are being skipped over for debugging crd updates rather than skipping silently.

Although the goal is to have useful debug statements to surface more context and not just pollute the debug logs so I'm open to rewording or removing them if you see fit.